src/tls_wrap.cc

```diff
@@ -90,8 +90,10 @@ TLSWrap::~TLSWrap() {
   MakePending();
 
   // And destroy
-  while (WriteItem* wi = pending_write_items_.PopFront())
+  while (WriteItem* wi = pending_write_items_.PopFront()) {
+    wi->w_->Done(UV_ECANCELED);
```

Comment:
Also, is there a reason why this can't be replaced with a call to `InvokeQueued(UV_ECANCELED)`?